packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.5.0;
```

Comment:
Why use the `0.5` compiler instead of a higher version (for both this contract and `OfferableIdentity`)?
I'm not saying it should be higher, I'm not too sure. But it maybe a higher version would mean a contract inheriting this contract could use new features?

Reply:
Good idea. I'll upgrade version